src/axom/slic/interface/slic_macros.hpp

```diff
@@ -72,6 +72,20 @@
     }                                                               \
   } while(axom::slic::detail::false_value)
 
+/*!
+ * @brief Macro that logs given error message only on root.
+ * \note Must call `axom::slic::initialize(shouldLogRootMessages=true)`
+ *   or set via `axom::slic::setRoot(true)`. Otherwise, this not filter based on root.
+ */
+#define SLIC_ERROR_ROOT(msg) SLIC_ERROR_IF(axom::slic::isRoot(), msg)
```

Comment:
Gut check: Will this cause all ranks to fail, or just the root?

We'd want the root to log, but all ranks to fail.

Reply:
Yea that doesn't currently happen.  Let me see if I can rework the macro from the naive solution to make that happen.